public/app/plugins/datasource/loki/components/LokiLabelBrowser.tsx

```diff
@@ -187,29 +187,23 @@ export class UnthemedLokiLabelBrowser extends React.Component<BrowserProps, Brow
   };
 
   onClickRunLogsQuery = () => {
-    reportInteraction('grafana_loki_log_browser_closed', {
+    reportInteraction('grafana_loki_label_browser_closed', {
       app: this.props.app,
-      closeType: 'showLogsButton',
```

Comment:
Wait, is there reason why we are removing these?

Reply:
I think it would be interesting to keep them. And for modal close, we can always just add something like `closeType: modalClose`, if we want to have it.

Reply:
@ivanahuckova this makes much more sense, sorry about that. 👀